core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -102,3 +103,9 @@ case object KAFKA_0_10_0_IV0 extends ApiVersion {
   val messageFormatVersion: Byte = Message.MagicValue_V1
   val id: Int = 4
 }
+
+case object KAFKA_0_10_0_IV1 extends ApiVersion {
+  val version: String = "0.10.0-IV1"
+  val messageFormatVersion: Byte = Message.MagicValue_V1
+  val id: Int = 4
```

Comment:
About using KAFKA_0_10_0 as the last version. The initial intention is to make trunk always in a releasable state by pointing KAFKA_0_10_0 to the last IV tag. I guess Gwen's suggestion is to only add KAFKA_0_10_0 just before the release and use it to replace the last IV tag? That could also work, but would mean that we need to remember doing this before every release and also make last minute code changes.